community/python-gssapi/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Dmitry Romanenko <dmitry@romanenko.in>
+# Maintainer: Dmitry Romanenko <dmitry@romanenko.in>
+pkgname=py-gssapi
+_pkgname=gssapi
+pkgver=1.5.1
+pkgrel=0
+pkgdesc="A Python interface to RFC 2743/2744 (plus common extensions)"
+url="https://github.com/pythongssapi/python-gssapi"
+arch="all"
+license="ISC"
+depends=""
+#checkdepends="py-nose py-shouldbe py-k5test" # Tests are missing alpine packages
+makedepends="python2-dev python3-dev krb5-dev py-setuptools"
+subpackages="py3-$_pkgname:_py3 py2-$_pkgname:_py2"
+source="https://files.pythonhosted.org/packages/source/g/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
```

Comment:
Do you require python2 support for some reason? Python2 is soon eol, so we discourage new packages being built for python2.

Reply:
Honestly, I just used the typical metapackage template. If this is requirement from now on to only publish python3 packages, no problem, I'll adjust

Reply:
`newapkbuild -y` is already a python3 only template.